package/crossplane.yaml

```diff
@@ -4,6 +4,6 @@ kind: Function
 metadata:
   name: function-appcat
 spec:
-  image: ghcr.io/vshn/appcat:latest
+  image: ghcr.io/vshn/appcat:nonslaalerts
```

Comment:
Can you please reset this? It might break other stuff down the road.

If you use the make target with the `IMG_TAG` variable, you don't have to change this, as Make will inject it during the build: https://github.com/vshn/appcat/blob/master/package/package.mk#L3

Reply:
reverted